docs/ref/request-response.txt

```diff
@@ -293,16 +293,23 @@ Methods
 
     Example: ``"/minfo/music/bands/the_beatles/?print=true"``
 
-.. method:: HttpRequest.build_absolute_uri(location)
+.. method:: HttpRequest.build_absolute_uri(location=None)
 
     Returns the absolute URI form of ``location``. If no location is provided,
     the location will be set to ``request.get_full_path()``.
 
     If the location is already an absolute URI, it will not be altered.
     Otherwise the absolute URI is built using the server variables available in
-    this request.
+    this request. For example:
```

Comment:
Did you want to use `::` and indent here?

e.g. https://github.com/django/django/blob/d6df777ccc4ad3160d3efefb1cd182c219877840/docs/ref/request-response.txt#L388-L394

Reply:
I copied `QueryDict.__init__` - lines starting `>>>` automatically start code blocks in rST/Sphinx, new to me too! Happy to convert these if you think it's better to be consistent.